### PR TITLE
BZ 1591637 - Add note about mount options

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1591637

### CODE REVIEW

**Comment by weherdh** [comment]
LGTM for azure file part. But I am not sure for the local volume. @qinpingli Could you please help to take a look? Thanks.

**Comment by qinpingli** [comment]
LGTM

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

File: install_config/configuring_local.adoc [hunk 1]
```diff
@@ -159,6 +159,45 @@ $ oc new-app -p CONFIGMAP=local-volume-config \
   -p SERVICE_ACCOUNT=local-storage-admin \
   -p NAMESPACE=local-storage local-storage-provisioner
 ----
+
+ifndef::openshift-origin[]
+<1> Replace `v3.9` with the right {product-title} version.
```
Comment (kalexand-rh) [inline install_config/configuring_local.adoc]: Where is "v3.9" to replace? I don't see the other half of this callout.

File: install_config/configuring_local.adoc [hunk 2]
```diff
@@ -159,6 +159,45 @@ $ oc new-app -p CONFIGMAP=local-volume-config \
...
+[source, yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+ name: local-hdd
+provisioner: kubernetes.io/no-provisioner
+volumeBindingMode: WaitForFirstConsumer
+----
++
```
Comment (kalexand-rh) [inline install_config/configuring_local.adoc]: I'd remove this line.

File: install_config/configuring_local.adoc [hunk 3]
```diff
@@ -159,6 +159,45 @@ $ oc new-app -p CONFIGMAP=local-volume-config \
...
+
+. Add the necessary storage classes:
++
+[source, bash]
+----
+oc create -f ./storage-class-ssd.yaml
+oc create -f ./storage-class-hdd.yaml
+----
++
+.storage-class-ssd.yaml
```
Comment (kalexand-rh) [inline install_config/configuring_local.adoc]: I'm pretty against putting the yaml after the command to create the object that uses the yaml.

File: install_config/persistent_storage/persistent_storage_azure_file.adoc [hunk 4]
```diff
@@ -42,6 +42,11 @@ $ /usr/sbin/setsebool -P virt_use_samba on
 $ /usr/sbin/setsebool -P virt_sandbox_use_samba on
 ----
 
+[NOTE]
+====
+Mount options is not available in {product-title} 3.7.43. You should not use Azure File in {product-title} 3.7.43. You can upgrade to {product-title} 3.7.46 or later to use Azure File.
```
Comment (kalexand-rh) [inline install_config/persistent_storage/persistent_storage_azure_file.adoc]: s/is/are
s/should not/cannot
I'd say "To use Azure File, upgrade to {product-title} 3.7.46 or later."

I'd double-check the conditionalization of this note.

**Pull request merged**